app/include/lwip/tcp_impl.h

```diff
@@ -130,7 +130,7 @@ u32_t            tcp_update_rcv_ann_wnd(struct tcp_pcb *pcb)ICACHE_FLASH_ATTR;
 #define TCP_OOSEQ_TIMEOUT        6U /* x RTO */
 
 #ifndef TCP_MSL
-#define TCP_MSL 60000UL /* The maximum segment lifetime in milliseconds */
+#define TCP_MSL 5000UL /* The maximum segment lifetime in milliseconds */
```

Comment:
Maybe leave a comment there indicating _why_ this was changed so next time lwIP is updated it shows up in the diff. Can even include a reference to the GitHub issue number. If we don't document such changes there's a real risk that subsequent SDK updates will revert them w/o us noticing.